test/Ocelot.AcceptanceTests/MapRequestTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.AcceptanceTests
+{
+    public class MapRequestTests : IDisposable
```

Comment:
Cannot get it! What class do we test here? `RequestMapper`?
Better to rename to `ChunkedEncodingTests` ❗

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1972/commits/dcbbab7119180345afaf001fbf3664f64b2fafd7 ✔️